posts/2018-09-04-do-not-use-list-monoid.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: PSA – Don't use the List monoid
```

Comment:
I would favour a title that sounds less like "List considered harmful" ☺

Reply:
Any suggestions? :)

Reply:
Maybe "Chains: A more efficient data structure for concatenation"?

Reply:
Went with a different name that's still somewhat click-baity, but conveys the message well IMO :)